rca/static_src/sass/components/_section.scss

```diff
@@ -387,6 +437,26 @@
                     grid-column: 1 / span 3;
                 }
             }
+
+            &::before {
+                #{$root}__notch--opposite & {
+                    right: calc((var(--margin-width) * -1)); // Make negative
+                    width: calc(var(--margin-width));
+
+                    @include media-query(large) {
+                        right: calc(
+                            (
+                                    var(--margin-width) + var(--grid-width) +
+                                        var(--outer-grid-width)
+                                ) * -1
```

Comment:
nipick: formatting looks interesting here - guessing it's the line length rule?

Reply:
hah yea, awful isn't it, would you put an ignore in here?

Reply:
i think if it saved-on-format that way then i'd just leave it as is